src/aws/osml/tile_server/viewpoint/routers.py

```diff
@@ -33,6 +37,8 @@
 )
 from .queue import ViewpointRequestQueue
 
+logger = logging.getLogger("uvicorn")
```

Comment:
I don't think we need this logger here.

Reply:
Yep. This shouldn't be here; fix incomming. I think I added it temporarily when doing some debugging.

As a side note I noticed that there is a logger setup as a member variable on the ViewpointRouter class and that it is tied to "uvicorn.access". That seems odd to me but I didn't have a chance to dig into and understand why logging is setup that way for this class.